docker/minimal-spec/puppetserver-minimal_spec.rb

```diff
@@ -0,0 +1,49 @@
+#! /usr/bin/env ruby
+
+require 'rspec/core'
+require 'open3'
+require 'pupperware/spec_helper'
+
+describe 'puppetserver container' do
+  include Pupperware::SpecHelpers
+
+  before(:all) do
+    run_command('docker pull puppet/puppet-agent-ubuntu:latest')
+    if ENV['PUPPET_TEST_DOCKER_IMAGE'].nil?
+      fail <<-MSG
+      error_message = <<-MSG
+  * * * * *
+  PUPPET_TEST_DOCKER_IMAGE environment variable must be set so we
+  know which image to test against!
+  * * * * *
+      MSG
+		end
```

Comment:
super minor nit... looks we have some tabs interspersed with spaces on a few lines here